sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -3,16 +3,15 @@
 ## 5.1.0b1 (Unreleased)
 
 **New features**
-- We are now targeting the service's v3.1-preview.1 API as the default. If you would like to still use version v3.0 of the service,
+- We are now targeting the service's v3.1-preview API as the default. If you would like to still use version v3.0 of the service,
```

Comment:
this could be confusing to the user, especially that you referred to the concrete preview version before in older changelog notes [here](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md#100b6-2020-05-27).

Reply:
that's fair, but I think with these two new versions, it's unnecessary to support both, and once the service GAs, we're also only going to support v3.0 and v3.1, so I'm comfortable squashing v3.1-preview.1 and v3.1-preview.2 into one api version

Reply:
my comment is not about supporting both but rather about being consistent with the service naming of the endpoints. Otherwise the user would wonder what is v3.1-preview because no such version exist in the service docs.

Reply:
I see your point, @kristapratico and I had discussed this, and I think we both overall prefer exposing it as v3.0 and v3.1-preview instead of v3.1-preivew.2. Since we've never actually done a release on our side targeting the v3.1-preview.1 API, I think it would be weird to come out directly with a v3.1-preview.2, and I think it overall reduces some overhead for the user exposing it as v3.1-preview version instead of specifically v3.1-preview.2 in all of the docstrings.

Reply:
@deyaaeldeen what I meant by overhead was having to explicitly now that this was the second preview version. Since there's no reason for a user to discern between 3.1-preview.1 and 3.1-preview.2, I think it would be cleaner to just combine them as 3.1-preview